docs/docs/gatsby-starters.md

```diff
@@ -0,0 +1,64 @@
+---
+title: 'Gatsby Starters'
+---
+
+GatsbyJS starters are premade projects packed with features which can be used for accelerating development speed. They are also a great source for learning about implementations of different features.
+
+Using a starter:
+
+```sh
+# Install gatsby-cli before beginning
+gatsby new STARTER_URL YOUR_PROJECT
+
+# Example
+gatsby new https://github.com/gatsbyjs/gatsby-starter-blog my-log
+```
+
+Currently available starters:
```

Comment:
Kill this line. It's redundant because if a starter isn't here it isn't currently available ;-)